### ISSUE TITLE
Accessibility - Resources Screen - TalkBack User Can not Activate Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-Android#27]
As a TalkBack user, I cannot activate the two DOL.gov links by double-tapping on them.

After playing around, I found they gave you an option to open the links if you double-tapped and then held the link. My guess this might be an Android thing because it doesn't happen that way in iOS, but I have not seen this before in any other Android testing.

**Comment by greg-gruse** [comment]
The ability to click on email or url links require the user to first swipe up then right.....  Please see reference below:

As mentioned in the Google Support page, the user will need to access the Local Context Menu via a TalkBack gesture (default gesture is Swipe up then right) to activate a TextView link.     https://support.google.com/accessibility/android/answer/6378148?hl=en

**Comment by jfc3-dol** [comment]
I was able to activate the links after is buzzed/chimed by double tapping and then holding to get a pop-up asking which browser to use in the case of the link and the if I wanted to use Gmail for the e-mail address. I previously don't remember having to hold to get options and only needing to double-tap to get the link to work in Android.

Going to close this issue.
